schema-registry-mock/src/main/java/com/bakdata/schemaregistrymock/SchemaRegistryMock.java

```diff
@@ -310,6 +310,11 @@ public String getName() {
         }
     }
 
+    private static String removeQueryParameters(final String url) {
+        final int index = url.indexOf("?");
+        return index == -1? url : url.substring(0, index);
```

Comment:
```suggestion
        return index == -1 ? url : url.substring(0, index);
```

Reply:
Is your brain working as a code formatter or did you use your IDE for review? :P